cpp/src/arrow/CMakeLists.txt

```diff
@@ -534,7 +534,7 @@ else()
 endif()
 
 if(ARROW_BUILD_BUNDLED_DEPENDENCIES)
-  string(APPEND ARROW_PC_LIBS_PRIVATE " -larrow_bundled_dependencies")
+  set(ARROW_STATIC_INSTALL_INTERFACE_LIBS_REAL ${ARROW_STATIC_INSTALL_INTERFACE_LIBS})
```

Comment:
Why do we need `_REAL` alias?
Is there any problem if we use `ARROW_STATIC_INSTALL_INTERFACE_LIBS` directly?